tools/fixExtensions.go

```diff
@@ -28,8 +29,6 @@ func main() {
 		fmt.Printf("\nProcessing batch %d to %d: ", batch, batch+100)
 		rows, err := db.Query(`SELECT id, raw_cert
 					FROM certificates
-					WHERE x509_certificatepolicies IS NULL OR x509_certificatepolicies='null'
-					   OR permitted_names IS NULL OR permitted_names='null'
```

Comment:
Oh, right. I was looking at the batch numbers and that was progressing, so I assumed it was fine. I think we could just ORDER BY id and OFFSET by the processed amount and LIMIT to the batch size. Thoughts?